Sources/PartialSheet/PartialSheetViewModifier.swift

```diff
@@ -32,13 +32,18 @@ struct PartialSheet: ViewModifier {
     
     /// The offset for the drag gesture
     @State private var dragOffset: CGFloat = 0
-    
+
     /// The point for the top anchor
     private var topAnchor: CGFloat {
-        return max(presenterContentRect.height +
+        let calculatedTop = presenterContentRect.height +
             (UIApplication.shared.windows.first?.safeAreaInsets.bottom ?? 0) -
-            sheetContentRect.height - handlerSectionHeight,
-                   style.minTopDistance)
+            sheetContentRect.height - handlerSectionHeight
+        
+        if calculatedTop > self.style.minTopDistance {
+            return calculatedTop
+        }
+        
+        return self.style.minTopDistance
```

Comment:
```
let calculatedTop = presenterContentRect.height +
              (UIApplication.shared.windows.first?.safeAreaInsets.bottom ?? 0) -		              (UIApplication.shared.windows.first?.safeAreaInsets.bottom ?? 0) -
              sheetContentRect.height - handlerSectionHeight,		              sheetContentRect.height - handlerSectionHeight
                     style.minTopDistance)		 
if calculatedTop > self.style.minTopDistance {
              return calculatedTop
          }
 
return self.style.minTopDistance
```

Can we change the if with a guard?
Is the self really needed?

Reply:
Sure, we can change it.

```self``` is there to express scope but I am not religious about it :-) I will update the PR